docs/docs/cli/generated/tracetest_environment.md

```diff
@@ -0,0 +1,34 @@
+# CLI Referece
+## tracetest environment
+
+Manage your tracetest environments
```

Comment:
(Comment for all "base" commands: `datastore`, `environment`, `server` and `test`)

Do you think we should add any comment saying this command does nothing if it is called directly? Or maybe we could turn to add more docs in the line of `helm repo --help`?

<img width="606" alt="image" src="https://user-images.githubusercontent.com/12397879/216724207-8d23997e-b548-42a5-b9c0-2c525948ed53.png">